src/test/scala/org/hammerlab/guacamole/commands/SomaticStandardCallerRealDataSuite.scala

```diff
@@ -117,21 +118,21 @@ class SomaticStandardCallerRealDataSuite
 
   test("testing simple negative variants on syn1") {
     val (tumorReads, normalReads) =
-      TestUtil.loadTumorNormalReads(
+      loadTumorNormalReads(
         sc,
         "synthetic.challenge.set1.tumor.v2.withMDTags.chr2.syn1fp.sam",
         "synthetic.challenge.set1.normal.v2.withMDTags.chr2.syn1fp.sam"
       )
 
     val negativePositions = Array[Long](
       216094721,
-      3529313,
-      8789794,
+        3529313,
```

Comment:
yea i right-aligned a few large lists of loci integers like this is a few places, thought it was clearer